src/Host/Client/Impl/Definitions/IRSessionProvider.cs

```diff
@@ -9,6 +9,7 @@
 namespace Microsoft.R.Host.Client {
     public interface IRSessionProvider : IDisposable {
         Uri BrokerUri { get; }
+        string BrokerName { get; }
```

Comment:
Ok, I'll rename IRHostConnector interface into IRBroker

---
In reply to: [78037959](https://github.com/Microsoft/RTVS/pull/2268#discussion_r78037959) [](ancestors = 78037959)